django_mailbox/models.py

```diff
@@ -578,6 +611,23 @@ def html(self):
             self.get_email_object(), 'text', 'html'
         ).replace('\n', '').strip()
 
+    #rulzart - added a quick way to get Cc addresses
+    @property
+    def get_cc(self):
```

Comment:
``get_`` prefix is incoherent for other properties in model. I suggest remove ``get_``.